rmp-serde/src/lib.rs

```diff
@@ -79,6 +79,8 @@ pub mod config;
 pub mod decode;
 pub mod encode;
 
+pub const MSGPACK_EXT_STRUCT_NAME: &'static str = "_ExtStruct";
```

Comment:
Could you add a small doc comment here to mention this constant's expected usage?

I imagine just having `# Example` and then copying the code in from one of the `ExtStruct` tests could be useful enough.